docs/operating-scylla/procedures/cluster-management/arbiter-dc.rst

```diff
@@ -0,0 +1,50 @@
+=========================================================
+Preventing Quorum Loss in Symmetrical Multi-DC Clusters
+=========================================================
+
+ScyllaDB requires at least a quorum (majority) of nodes in a cluster to be up
+and communicate with each other. A cluster that loses a quorum can handle reads
+and writes of user data, but cluster management operations, such as schema and
+topology updates, are impossible.
```

Comment:
Maybe we could mention at the end of the paragraph that if the quorum is permanently lost, performing [Manual Recovery Procedure](https://opensource.docs.scylladb.com/stable/troubleshooting/handling-node-failures.html#recovery-procedure) is the only way to bring the cluster back to full functionality.